`make install` not working

Commits d992a90affb95b4246df5f9cd7b720a4094d0f3d and 7d7271caa0d1c415f6c1088716627a0d0c29224d tried to fix a build issue with 0734503d20bb42f2687b757c89cf0835a4c4b7e2, but it missed a few files, specifically:

```
11:29 ➜  cerberus git:(0734503d2) git checkout d992a90affb95b4246df5f9cd7b720a4094d0f3d
Previous HEAD position was 0734503d2 (cerb_runtime) disabling local dune runtime discovery
HEAD is now at d992a90af (vip) moving vip to the cerberus-lib package
11:29 ➜  cerberus git:(d992a90af) make && make cn && make install && make install_cn
[DUNE] cerberus
(debug 0): constructValue_aux: is WRONG for union ==> always assigning the first member
[DUNE] cn

DONE
[DUNE] cerberus
[DUNE] cerberus-lib
[DUNE] install cerberus-lib
[DUNE] install cerberus
Error: The following files which are listed in
_build/default/cerberus.install cannot be installed because they do not
exist:
- _build/install/default/lib/cerberus/META
- _build/install/default/lib/cerberus/dune-package
- _build/install/default/lib/cerberus/opam
- _build/install/default/lib/cerberus/runtime/libc/libc.co
- _build/install/default/lib/cerberus/runtime/libc/libc_inner_arg_temps.co
- _build/install/default/lib/cerberus/runtime/libc/libm.co
- _build/install/default/lib/cerberus/runtime/libc/src/LICENSE
- _build/install/default/lib/cerberus/runtime/libc/src/ctype.c
- _build/install/default/lib/cerberus/runtime/libc/src/internal.c
- _build/install/default/lib/cerberus/runtime/libc/src/internal.h
- _build/install/default/lib/cerberus/runtime/libc/src/locale.c
- _build/install/default/lib/cerberus/runtime/libc/src/math.c
- _build/install/default/lib/cerberus/runtime/libc/src/signal.c
- _build/install/default/lib/cerberus/runtime/libc/src/stat.c
- _build/install/default/lib/cerberus/runtime/libc/src/stdio.c
- _build/install/default/lib/cerberus/runtime/libc/src/stdlib.c
- _build/install/default/lib/cerberus/runtime/libc/src/string.c
- _build/install/default/lib/cerberus/runtime/libc/src/time.c
- _build/install/default/lib/cerberus/runtime/libc/src/uio.c
- _build/install/default/lib/cerberus/runtime/libc/src/unistd.c
- _build/install/default/lib/cerberus/runtime/libc/src/utime.c
- _build/install/default/lib/cerberus/runtime/libc/src/vfscanf.c
- _build/install/default/bin/cerberus
- _build/install/default/bin/cerberus-vip
- _build/install/default/doc/cerberus/LICENSE
- _build/install/default/doc/cerberus/README-cheri.md
- _build/install/default/doc/cerberus/README.md
```

I thought the issue is an out of date `cerberus.install` file, so I tried deleting it (it's auto-generated, and perhaps this should be done by `make clean`) . This leads to a suggestion by dune, which succeeds, but regenerates the same erroneous cerberus.install file.

```
11:42 ➜  cerberus git:(main) make && make install
[DUNE] cerberus
[DUNE] cerberus
[DUNE] cerberus-lib
[DUNE] install cerberus-lib
[DUNE] install cerberus
Error: The following <package>.install are missing:
- _build/default/cerberus.install
Hint: try running 'dune build [-p <pkg>] @install'
make: *** [Makefile:304: install] Error 1
11:42 ➜  cerberus git:(main) dune build cerberus @install
zsh: correct 'build' to '_build' [nyae]? n
Error: Don't know how to build cerberus
11:43 ➜  cerberus git:(main) dune build -p cerberus @install
(cd _build/default/runtime/libc && ../../../install/default/bin/cerberus --nolibc -I include -I include/posix --sequentialise --rewrite -o libm.co src/math.c)
(debug 0): constructValue_aux: is WRONG for union ==> always assigning the first member
```

Fwiw, `make`, `make cn` and `make install_cn` all still work fine, except on the Docker builds, which may be a separate issue? https://github.com/rems-project/cerberus/actions/runs/13727050176/job/38395817125#step:6:5189

```
  > [linux/amd64 10/14] RUN eval `opam env`   && make install_cn:
Warning: RNING] Running as root is not recommended
0.101 [DUNE] cerberus-lib
0.181 File "sibylfs/dune", line 11, characters 55-62:
0.181 11 |   fs_prelude fs_printer fs_spec lem_support list_array sibylfs)
0.181                                                             ^^^^^^^
0.181 Error: Module Sibylfs doesn't exist.
0.218 make: *** [Makefile:294: cerberus-lib] Error 1
------
Dockerfile.ubuntu:17
--------------------
  16 |     
  17 | >>> RUN eval `opam env` \
  18 | >>>   && make install_cn
  19 |     
--------------------
```

--- a/Makefile
+++ b/Makefile
@@ -299,7 +299,7 @@ install_lib: cerberus-lib
 	$(Q)dune install cerberus-lib
 
 .PHONY: install
-install: cerberus install_lib
+install: install_lib cerberus
 	@echo "[DUNE] install cerberus"
 	$(Q)dune install cerberus
 

